sdk/storage/azure-storage-blob-batch/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.11.0-beta.2 (Unreleased)
-
+## 12.11.0 (2021-09-15)
+- GA release
```

Comment:
Maybe `This release contains bug fixes to improve quality.` like we do in .net when we absorb dependency updates. See https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Common/CHANGELOG.md for example.

Reply:
Does it contain bug fixes to improve quality? It's just a release to move features previously released as beta to stable

Reply:
through dependency update of core and blobs perhaps. Not many features in beta.